jbmc/src/java_bytecode/assignments_from_json.cpp

```diff
@@ -67,9 +67,8 @@ static java_class_typet
 followed_class_type(const exprt &expr, const symbol_table_baset &symbol_table)
 {
   const pointer_typet &pointer_type = to_pointer_type(expr.type());
-  const java_class_typet &java_class_type = to_java_class_type(
-    namespacet{symbol_table}.follow(pointer_type.base_type()));
-  return java_class_type;
+  return to_java_class_type(namespacet{symbol_table}.follow_tag(
+    to_struct_tag_type(pointer_type.base_type())));
```

Comment:
❓ Are these types always tags?

Reply:
Yes, the Java front-end makes sure that those are always tags.